### DISCUSSION TITLE
Add option to delay splash screen so we add music and such

### DISCUSSION THREAD

**Discussion by newriderca** [EdgeTX/edgetx#2536] [Ideas]
Please developer give us option to delay splash screen so we add music and such.

**Comment by pfeerick** [comment]
It's best to not make multiple posts asking for the same thing... usually just ends up annoying people, and getting all but one closed ;) Since you already opened the issue, https://github.com/EdgeTX/edgetx/issues/2537, lets continue the discussion there for now.
